docs/python/convertDoxygen.py

```diff
@@ -106,11 +106,13 @@
     # Loop through module list and create a Writer for each module to 
     # load and generate the doc strings for the specific module
     for moduleName in moduleList:
+        if not moduleName:
+            continue
         writer = Writer(packageName, moduleName)
         # Parser.traverse builds the docElement tree for all the 
         # doxygen XML files, so we only need to call it once if we're
         # processing multiple modules
-        if (docList == None):
+        if (docList is None):
```

Comment:
using `is` instead of `==` is considered idiomatic because `None` is a singleton.